docs/about_fairness.html

```diff
@@ -109,6 +110,10 @@ <h2>FNR Difference<a class="headerlink" href="#fnr-difference" title="Permalink
 <div class="math notranslate nohighlight">
 \[E[d(X)=0 \mid Y=1, g(X)] = E[d(X)=0, Y=1]\]</div>
 </section>
+<section id="for-difference">
```

Comment:
I see that you changed the HTML file.. but here is how it works:

- We need to make this change in "docsrc" folder, inside the .rst files.
- These document sources are then compiled into the html files that we see in the docs folder
- The html are what's displayed in the github IO pages 

So "docs" are just outputs and our documentation is in the "rst" files. Can you revert changes to this file, and make the edits on the doc source rst file instead? 

Then, there is a "sphinx" command that we need to run on the docsrc folder.. which updates the htmls in the "docs" folder.. You might need to google how this command works. If not, @takojunior  can help re-generate the documents for you

Reply:
My most recent 2 commits should address this; I reverted the changes to the HTML files, made them instead to the .rst files, and generated everything with sphinx-build

Reply:
The PR and recent commits look good to me. Some minor comments are

- I see CHANGELOG.txt may need an update, e..g add FOR to Fairness metrics
- The _version file under jurity folder may need update from 1.3.3 to 1.3.4.
- README file may also add the FOR difference

I assume after we update the version, we need to rebuild the doc files. Let me know if you need some of my helps on this. Then we will have a new Pypi release I think, but we can take care of it after the merge.

Reply:
In a new commit, I addressed these points and rebuilt the docs, but I think the change in version ended up not requiring a rebuild because the one I ran didn't change any files. Please let me know if there's anything else I can do.

Reply:
Thank you!